package.json

```
@@ -137,10 +138,10 @@
     "@testing-library/jest-dom": "^5.14.1",
     "@testing-library/react": "^14.0.0",
     "@testing-library/user-event": "^14.4.3",
-    "@utrecht/component-library-css": "^1.0.0-alpha.526",
-    "@utrecht/component-library-react": "^1.0.0-alpha.353",
-    "@utrecht/components": "^1.0.0-alpha.473",
-    "@utrecht/design-tokens": "^1.0.0-alpha.505",
+    "@utrecht/component-library-css": "^1.0.0-alpha.604",
+    "@utrecht/component-library-react": "^1.0.0-alpha.413",
+    "@utrecht/components": "^1.0.0-alpha.565",
+    "@utrecht/design-tokens": "^1.0.0-alpha.597",
```

something I noticed working on my own branch - these dependencies must be pinned instead of allowing a range (went wrong in my earlier commit):

```suggestion
    "@utrecht/component-library-css": "1.0.0-alpha.604",
    "@utrecht/component-library-react": "1.0.0-alpha.413",
    "@utrecht/components": "1.0.0-alpha.565",
    "@utrecht/design-tokens": "1.0.0-alpha.597",
```